Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner() {
+    return (
+        <img src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt=""/>
```

Comment:
please use the alt attribute so that when the image does not load text is displayed and also screen readers will output that text for people using screen readers